common/lib/capa/capa/templates/clarification.html

```diff
@@ -0,0 +1,4 @@
+<span class="clarification">
+	<i data-tooltip="${clarification | h}" data-tooltip-show-on-click="true" class="fa fa-info-circle"></i>
```

Comment:
@sarina, I always add an ```aria-hidden="true"``` to icon fonts that are adjacent to accessible text (like line 3) because screen readers will focus on them and not hear anything, and screen readers that also visually display what is being spoken for those with cognitive disabilities (like voiceover on Mac) display a generic glyph.